pkg/network/ebpf/c/protocols/classification/shared-tracer-maps.h

```diff
@@ -4,11 +4,15 @@
 #include "map-defs.h"
 #include "port_range.h"
 #include "protocols/classification/stack-helpers.h"
+#include "protocols/tls/tls.h"
 
 // Maps a connection tuple to its classified protocol. Used to reduce redundant
 // classification procedures on the same connection
 BPF_HASH_MAP(connection_protocol, conn_tuple_t, protocol_stack_wrapper_t, 0)
 
+// Map to store extra information about TLS connections like version, cipher, etc.
+BPF_HASH_MAP(tls_enhanced_tags, conn_tuple_t, tls_info_wrapper_t, 1)
```

Comment:
I don't think the change to be brought to USM, but to be limited to the NPM classification part
If USM is enabled without NPM, we will not present in the UI the tags you're adding

Reply:
sounds good I will move this to the NPM map defs